### PR TITLE
Add an A/B test for quick responses in the Help Center

### PR DESCRIPTION
**Pull request by sirino**
## Proposed Changes
I am adding an A/B test to disbale the "Quick Response" screen when a user contacts support through the Help Center. Users in the `no_quick_response` group will not see a quick response before they are able to email or chat with support.

## Testing Instructions

1. Use this live branch
2. Assign yourself to the experiment `calypso_helpcenter_quick_response_deflection_rate` as a user in the `control` group.
3. On a site with a paid plan, click on the "?" in the top right of your dashboard to open the help center
4. Enter a question of your choice and you will be shown a list of recommended resources. Click on the 'Still need help' button.
5. Choose "Live chat" from the list of options to contact support. You may need to wait a while or try again if there is no availability.
6. On the contact from, click 'Continue'.
7. Assert that your are shown a "Quick response" or, if none can be retrieved, a message that says so.
8. Assert that you can click 'Continue' and a chat box opens. Let them know that you're testing.
9. Go back to step 5 and choose 'Email'. Repeat steps 6 and 7. Assert that the 'Continue' button is blue but **DO NOT** submit an email. Doing so will prevent you from being able to test this step again.
10. Assign yourself to the experiment `calypso_helpcenter_quick_response_deflection_rate` as a user in the `no_quick_response` group.
11. Repeat steps 3 to 6. Assert that you are not shown a "Quick response" and that the chat box opens right away.
12. Repeat step 9. You may test submitting an email now.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: packages/help-center/src/components/help-center-contact-form.tsx [hunk 1]
```diff
@@ -186,6 +187,15 @@ export const HelpCenterContactForm = () => {
 	const [ debouncedMessage ] = useDebounce( message || '', 500 );
 	const [ debouncedSubject ] = useDebounce( subject || '', 500 );
 
+	const [ , experimentAssignment ] = useExperiment(
+		'calypso_helpcenter_quick_response_deflection_rate'
+	);
+
+	if ( experimentAssignment?.variationName === 'no_quick_response' ) {
+		params.set( 'disable-gpt', 'true' ); // this is the one
```
Comment (AllTerrainDeveloper) [inline packages/help-center/src/components/help-center-contact-form.tsx]:
```suggestion
		params.set( 'disable-gpt', 'true' );
```

**[NON BLOCKER]** The comment (at least for me) has no meaningful impact. What about removing it or just setting it above the if condition, a bit more explained?

**Review by AllTerrainDeveloper — APPROVED** [review]
Code-wise looks good! But I'll wait for the live branch to be ready, seems somehow "broken"

File: packages/help-center/src/components/help-center-contact-form.tsx (hunk 1 above)
Comment (sirino) [reply packages/help-center/src/components/help-center-contact-form.tsx]: The comment shouldn't have been there anymore - removed it

**Comment by noahtallen** [comment]
Unfortunately, it didn't automatically post the calypso.live link, so I found it in the build log: https://container-zealous-golick.calypso.live/

**Comment by sirino** [comment]
Closing in favour of a pull request from a branch.

**Pull request closed without merging**